### ISSUE TITLE
Missing icons

### ISSUE THREAD

**Issue by dawidhyzy** [brianegan/font_awesome_flutter#28]
Google icons are missing. I used: `Icon(FontAwesomeIcons.googlePlus)`
and this is the result:
<img width="239" alt="screenshot 2019-01-22 at 10 07 59" src="https://user-images.githubusercontent.com/2484936/51515617-d9537e00-1e2d-11e9-91de-6ce22a116045.png">

**Comment by andrijasinski** [comment]
Hi there! I also found a missing icon: code-commit (https://fontawesome.com/icons?d=gallery&q=commit)

**Comment by brianegan** [comment]
Thanks for the heads up. I'll take a look!

**Comment by chabislav** [comment]
Hey guys, what version are you using? For some reason 8.2.0 doesn't work for me at all.  Can't find 'FontAwesomeIcons'.

**Comment by andrijasinski** [comment]
@chabislav I set a FontAwesomeFlutter to `any` - so far, so good.
`font_awesome_flutter: any`

**Comment by fliermanrw** [comment]
I encounter the same problem as @dawidhyzy. Anyone found a solution?

**Comment by brianegan** [comment]
Also, @andrijasinski -- It's a separate issue, but the icons you mention are Pro icons. At the moment, this library only supports the free icons offered by Font Awesome.

**Comment by andrijasinski** [comment]
@brianegan Yes, my bad! Thank you.

**Comment by brianegan** [comment]
@chabislav Hrm, that's super weird. I've just published `8.4.0` with the latest 5.7.0 Font Awesome Icons. Could you please try that version? As long as `flutter packages get` downloads the dependency and the import is references in the correct file, I don't think there should be any problem using the `FontAwesomeIcon` class!

**Comment by brianegan** [comment]
How's this one going for folks? Did uninstalling / reinstalling help? Latest version fix the issue? Thanks!

**Comment by brianegan** [comment]
Gonna close this out for now since I haven't heard back for a bit. Please let me know if ya need more help!

**Comment by anthonysette** [comment]
> If you're not seeing any icons at all, sometimes it means that Flutter has a cached version of the app on device and hasn't pushed the new fonts. I've run into that as well a few times...
> 
> Please try:
> 
> 1. Stopping the app
> 2. Running `flutter clean` in your app directory
> 3. Deleting the app from your simulator / emulator / device
> 4. Rebuild & Deploy the app.

It worked after just stopping the app, running `flutter clean` and rebuilding it. I didn't need to delete the app from the emulator. Thanks!

**Comment by fliermanrw** [comment]
> > If you're not seeing any icons at all, sometimes it means that Flutter has a cached version of the app on device and hasn't pushed the new fonts. I've run into that as well a few times...
> > Please try:
> > 
> > 1. Stopping the app
> > 2. Running `flutter clean` in your app directory
> > 3. Deleting the app from your simulator / emulator / device
> > 4. Rebuild & Deploy the app.
> 
> This worked for me after just stopping the app, running `flutter clean` and rebuilding it. I didn't need to delete the app from the emulator. Thanks!

Bit late, but same for me. Key in this is that it will not work when hot reloading the app. First time using the FontAwesome icons you'll need to stop and re-run your app (maybe even without Flutter clean).

**Comment by zyllus17** [comment]
> If you're not seeing any icons at all, sometimes it means that Flutter has a cached version of the app on device and hasn't pushed the new fonts. I've run into that as well a few times...
> 
> Please try:
> 
> 1. Stopping the app
> 2. Running `flutter clean` in your app directory
> 3. Deleting the app from your simulator / emulator / device
> 4. Rebuild & Deploy the app.

Did all the above but it is still not working on ^8.5.0

**Comment by eebeeDEV** [comment]
I had the same issue and even when running `Flutter clean`, still no icons were displayed.
Then I realized that this happend after I copied custom fonts in the web/asset/fonts/ folder and created a `fontmanifest.json` inside the asset folder.
After deleting both the fonts folder and `fontmanifest.json` file the icons re-appeared again.

I also noticed that after running `Flutter build web` with the custom fonts, the build creates the following folder structure /assets/**assets**/fonts/ 

Since then I've put the fonts folder at the root of the web `(/web/fonts/)` folder and the `fontmanifest.json` also and now both the custom fonts and the icons appear.
